qa/L0_batcher/test.sh

```diff
@@ -403,10 +399,12 @@ done
 # by comparing the "response send" timestamps.
 TEST_CASE=test_multi_batch_preserve_ordering
 
-if [[ $BACKENDS == *"custom"* ]]; then
+# FIXME sleep_for does not delay for expected time when passing 400ms 
```

Comment:
Here is the ticket https://jirasw.nvidia.com/browse/DLIS-2315
The execution delay works fine for system and no-shared memory but for cuda shared memory case I see an execution delay of 100 and 280ms instead of 100 and 400ms.